src/App.js

```diff
@@ -1,27 +1,22 @@
-import logo from './logo.svg';
+import React from 'react';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import { useSelector } from 'react-redux';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import AuthenticatedRoute from './routes/AuthenticatedRoute';
+import UnAuthenticatedRoute from './routes/UnAuthenticatedRoute';
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/MudasirSherwani/Book-Appointment-Front-End/assets/57421823/9410d8c4-8f96-423a-916a-61090ecceaad)

Reply:
![image](https://github.com/MudasirSherwani/Book-Appointment-Front-End/assets/86018834/1d2a7648-7027-472b-a553-e4838118b238)